fastlane/lib/fastlane/documentation/markdown_docs_generator.rb

```diff
@@ -185,7 +185,7 @@ def generate!(target_path: nil)
         FileUtils.cp(current_asset_path, File.join(docs_dir, "img", "actions", File.basename(current_asset_path)))
       end
 
-      UI.success("Generated new docs on path #{target_path}")
+      UI.success("Generated new docs files (actions.md, actions/*.md, mkdocs.yml, assets) on path #{target_path}")
```

Comment:
Maybe we should generate `(actions.md, actions/*.md, mkdocs.yml, assets)` from the actual changes instead of hardcoding them as a string

Reply:
I thought about building that string dynamically, adding an entry with each "step" - but that would just make the code more complicated for almost no benefit. Same for somehow using the actual changed files to generate this string out of it - super complicated for almost no use.

Or did I misunderstand? Is there a simple way I missed?

Reply:
I didn’t look at a way to implement it but I imagined the latter. I just noticed it as a future maintenance point and it doesn’t feel very “fastlaney”. But it’s also not really a problem either since this list isn’t likely to change often. 🤷🏻‍♀️

Reply:
I think I agree that this doesn't feel very "fastlaney". I _think_ I would lean towards keeping what was originally here and if someone is doesn't know and is curious what all got generated, they can just go look at that path 🙃

Reply:
Unfortunately not, as this is _inside_ a full clone of the docs where some additional files get added or modified.

Instead of one `UI.success` we could add an individual `UI.message` for each step (which would also make it consistent with the output for the assets).

Reply:
Also: fastlane is doing it this way in other places: https://github.com/fastlane/fastlane/blob/665a23e6351702bbd6980fe1f7ec75a6f1f7bfa6/fastlane/Fastfile#L392